config/initializers/content_security_policy.rb

```diff
@@ -17,7 +17,25 @@
   policy.frame_ancestors :self
 
   # Specify URI for violation reports
-  # policy.report_uri "/csp-violation-report-endpoint"
+  policy.report_uri lambda {
```

Comment:
What about to limit this to be set only when `ENV['DATADOG_CSP_API_KEY']` is present?

Reply:
so I initially did that, but I wanted the tests to run the lambda, otherwise we'd not know if there were exceptions in there

Reply:
Maybe the test can set ENV["DATADOG_CSP_API_KEY"] before it runs?

Reply:
how about this -- we run the lambda unconditionally (so dev/test naturally will go through the codepath), but we return `nil` if the `api_key.blank?`, so we don't set a useless `report_uri` ?

Reply:
Sure, sounds fine to me

Reply:
done!